src/main/java/com/revature/Flumblr/entities/Post.java

```diff
@@ -58,6 +59,7 @@ public class Post {
     private User user;
 
     @OneToMany(mappedBy = "post", orphanRemoval = true, fetch = FetchType.LAZY, cascade = CascadeType.ALL)
+    @OrderBy("createTime")
```

Comment:
let me know if I'm wrong but should it be "create_time" since that's the column name in the table?

Reply:
I thought it was the member name in the jpa class. It seemed to work without throwing errors when I ran it

Reply:
okay sounds good